src/agents/customer_group.jl

```diff
@@ -886,6 +889,7 @@ function solve_agent_problem!(
         @variable(Customer_PV_Storage_Opti, stor_charge[model_data.index_d, model_data.index_t] >= 0)
         @variable(Customer_PV_Storage_Opti, stor_discharge[model_data.index_d, model_data.index_t] >= 0)
         @variable(Customer_PV_Storage_Opti, stor_energy[model_data.index_d, model_data.index_t] >= 0)
+        @variable(Customer_PV_Storage_Opti, net_load_iv[model_data.index_d, model_data.index_t], Bin)
```

Comment:
I don't understand this variable name. Would something like `net_load_is_positive` or `net_load_sign` or `net_load_sign_indicator` make sense?